core/platform/transactions/gae_transaction_services.py

```diff
@@ -21,10 +21,14 @@
 
 import functools
 
+
```

Comment:
Any reason for using these blank lines here and below?

Reply:
Mistake. Fixed